### PR TITLE
HS-144: Updated Ingress, TLS, Keycloak, local install scripts, and more. 

### PR DESCRIPTION
**Pull request by jkapponms**
## Description
- Fixed the Ingress config
- Fixed several issues relating to Keycloak
- Script installing an apporpriate TLS cert to local clusters

## Jira link(s)
- https://issues.opennms.org/browse/HS-144

### CODE REVIEW

**Comment by jkapponms** [comment]
I don't know why my rebase picked up a bunch of changes in `platform`, `rest-server` and `shared-lib`

**Comment by jkapponms** [comment]
Gonna fix this on new branch and resubmit the PR.

**Comment by jkapponms** [comment]
Extra commits are because the HS-252 branch needs to be rebased.

**Comment by geraldhumphries** [comment]
Good stuff, does this PR allow Helm itself to install the opennms charts?

File: operator/go.mod [hunk 1]
```diff
@@ -9,16 +9,17 @@ require (
...
 	github.com/go-logr/logr v1.2.2
-	github.com/mittwald/go-helm-client v0.8.4
+	github.com/google/uuid v1.3.0
+	//github.com/mittwald/go-helm-client v0.8.4
+	github.com/operator-framework/api v0.15.0
 	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.1
 	go.uber.org/zap v1.20.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
-	helm.sh/helm/v3 v3.8.1
```
Comment (geraldhumphries) [inline operator/go.mod:17]: I noticed that the dependencies on helm and go-helm-client (plus the code that uses them) were removed. What's the plan regarding them?

Comment (jkapponms) [reply operator/go.mod:17]: No, separate ticket.

**Comment by jkapponms** [comment]
> Good stuff, does this PR allow Helm itself to install the opennms charts?

No, separate ticket.

**Review by geraldhumphries — APPROVED** [review]

File: ri.yaml [hunk 2]
```diff
@@ -0,0 +1,1395 @@
+apiVersion: apiextensions.k8s.io/v1
```
Comment (geraldhumphries) [inline ri.yaml:1]: Is this file "realm import"? I can't find any references to it, is it needed in this PR?

Comment (jkapponms) [reply ri.yaml:1]: leftover test file that I accidentally checked in

**Pull request merged**